### ISSUE TITLE
Create PR when epinio/epinio is released

### ISSUE THREAD

**Issue by jimmykarily** [epinio/helm-charts#21]
The current process is as this:

When epinio/epinio has a new release, [a cross CI hook](https://github.com/epinio/epinio/blob/main/.github/workflows/release.yml#L70-L76) triggers a new release of the epinio-helm-chart#epinio chart: https://github.com/epinio/epinio-helm-chart/blob/main/.github/workflows/release.yml#L7-L8
This only bumps the chart and app version so that the new docker image for the epinio server is used.

Sometimes this is not enough and we have to manually make more changes to the helm chart. It would be better if the automation simply created a PR with the same changes, but not release it.

We can still trigger a release on epinio-helm-chart when something is merged to main.

**Comment by juadk** [comment]
My plan is:
1. Remove auto release process for every PR we merge in `epinio/epinio-helm-chart`
2. We will release helm charts like we do for epinio, creating a tag and pushing it will trigger the release CI.
3. An Epinio release will open a PR in the helm chart git repo and in this PR we bump Helm Chart version
4. Remove version bump check in linting process (already the case if I'm not wrong)

And later, we can configure an installation test by ct only for the big helm chart (epinio-installer).
@jimmykarily is it ok for you?

**Comment by jimmykarily** [comment]
>     2. We will release helm charts like we do for epinio, creating a tag and pushing it will trigger the release CI.

What tag would that be? In the epinio-heml-chart repository we have 3 different chart with 3 different versions. Maybe release should be triggered manually then? I remember there was a way to run a GA pipeline manually, no?

**Comment by juadk** [comment]
Yes we can trigger it manually with `workflow_dispatch`. That means we will manually bump the version for every Helm charts when we release? (can be certainly automated later)

**Comment by juadk** [comment]
Addressed by #27

**Comment by juadk** [comment]
Next steps (install and release) are covered in #31

**Comment by olblak** [comment]
Is this issue still relevant? 
The release process is partially automated as we discussed so I propose to close this issue and open another if needed

**Comment by olblak** [comment]
After a discussion with @jimmykarily, we decided to close this issue as no improvement is desired at this stage of the project.
As a reminder, we automated helm chart content update when a release epinio or epinio-installer but we don't automate chart version update as we want to do it manually when we trigger an Epinio or an Epinio installer release.

Once we feel comfortable with automating helm chart releases, we'll continue working on this.
